src/lib/int.ts

```diff
@@ -202,6 +203,38 @@ class UInt64 extends CircuitValue {
     return new UInt64(z);
   }
 
+  /**
+   * Bitwise XOR gate on {@link UInt64} elements. Equivalent to the [bitwise XOR `^` operator in JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_XOR).
+   * A XOR gate works by comparing two bits and returning `1` if two bits differ, and `0` if two bits are equal. It applies XOR to all 64 bits of the elements.
+   * ```typescript
+   * let a = UInt64.from(5);    // ... 000101
+   * let b = UInt64.from(3);    // ... 000011
+   *
+   * let c = a.xor(b);    // ... 000110
+   * c.assertEquals(6);
+   * ```
+   */
+  xor(y: UInt64) {
+    return new UInt64(this.value.xor(y.value, UInt64.NUM_BITS));
+  }
```

Comment:
I think it would be useful to have the same method on `UInt32`! even if it's a bit redundant with the bit length, but the mathod kinda fits better on a bit-constrained type than on general field elements (not saying we should not also have it on Field)